docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -48,6 +49,326 @@ to connect to and issue SQL utilities, see the
 {docs-url}/client_installation/index.html[_{project-name} Client Installation Guide_].
 
 <<<
+[[cleanup_statement]]
+== CLEANUP Statement
+
+The CLEANUP Statement cleans up objects whose information is inconsistent in the metadata. This command can be used to clean up individual objects or all objects in a schema. This command can also be used to locate all inconsistent objects in the Trafodion metadata and then clean them up.
+
+Inconsistent objects can be cleaned up and removed from the metadata and HBase by using one of the following options.
+
+```
+CLEANUP {obj-type} {obj-name} [ ,UID {object-uid} ]

```

Comment:
I should have been more clear.

The Trafodion SQL Manual describes the case where there are multiple syntax items, where you must choose one. Notice in the GRANT example you mention that one has to supply either a list of privileges or the keywords "ALL [ PRIVILEGES ]". But in the CLEANUP example, there is just one choice, not multiple choices. I suppose one could argue that { a } means I have to choose a, but then so does specifying "a" by itself.